Plash/Menus.swift

```diff
@@ -158,10 +158,18 @@ extension AppState {
 	func updateMenu() {
 		menu.removeAllItems()
 
+		menu.addCallbackItem(
+			isManuallyDisabled ? "Enable" : "Disable"
+		) {
+			self.isManuallyDisabled.toggle()
+		}
+
+		menu.addSeparator()
+
 		if isEnabled {
 			addWebsiteItems()
 		} else {
-			menu.addDisabled("Deactivated While on Battery")
+			menu.addDisabled(isManuallyDisabled ? "Currently deactivated" : "Deactivated While on Battery")
```

Comment:
I don't think we need to show any text when it's manually disabled.

Reply:
Implemented in https://github.com/sindresorhus/Plash/pull/131/commits/88d6a797e22eab23d798e623b2aa92436bd7a10f